### PR TITLE
fix(MultiFunctionButton): Properly set width of smaller button, properly hide in-active button from screen-readers

### PR DESCRIPTION
**Pull request by lukelooker**
Result of design-research spike on MultiFunctionButton feedback. 

- Smaller of two buttons widths used within `MultiFunctionButton` will now be match size of the larger
- Hidden button will be `disabled` to prevent screen readers from focusing on it incorrectly

## Developer Checklist [ℹ️](https://github.com/looker-open-source/components/blob/main//CONTRIBUTING.md#developer-checklist)

- [x] ♿️ Accessibility addressed
- [x] 🖼 Image Snapshot coverage
- [x] 📚 Documentation updated
- [ ] 👾 Browsers tested
  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari

### CODE REVIEW

File: packages/components/src/List/index.ts [hunk 1]
```diff
@@ -30,4 +30,4 @@ export * from './ListItem'
 export * from './ListItemContext'
 export * from './ListItemLabel'
 export { listItemDimensions } from './utils'
-export { ListItemProps } from './types'
+export type { ListItemProps } from './types'
```
Comment (jhardy) [inline packages/components/src/List/index.ts:33]: just wanted to check this was meant to be included in this change?

**Review by jhardy — COMMENTED** [review]
All looks good and functions much better. Only one question on the `ListItemProps` change

File: packages/components/src/List/index.ts (hunk 1 above)
Comment (lukelooker) [reply packages/components/src/List/index.ts:33]: Completely unrelated to this PR. New PR opened to fix this separately #2463

File: packages/components/src/Layout/Semantics/Layout/Layout.story.tsx [hunk 2]
```diff
@@ -30,8 +30,8 @@ import { Heading } from '@looker/components/src'
 import {
   Constitution,
   ConstitutionShort,
-} from 'packages/components/src/__mocks__/Constitution'
-import { ItemsFiller } from 'packages/components/src/__mocks__/ListHelper'
+} from '../../../__mocks__/Constitution'
+import { ItemsFiller } from '../../../__mocks__/ListHelper'
```
Comment (lukelooker) [inline packages/components/src/Layout/Semantics/Layout/Layout.story.tsx:34]: Unrelated to this PR but noticed warnings about this when re-running image snapshots. Just including here for ease, no concerns regarding breakage.

**Comment by lukelooker** [comment]
@jhardy Went back and dealt with a few follow-up issues I found on my first take on this. Can you give it a final review?

**Comment by lukelooker** [comment]
FYI - PR has a couple of unrelated screenshots (Layout examples) that weren't added properly in the previous commit.

**Review by jhardy — APPROVED** [review]
👍

**Pull request merged**